curriculum/challenges/english/25-front-end-development/workshop-recipe-ingredient-converter/6735424ecfeb557d81dcc9d1.md

```diff
@@ -211,8 +200,12 @@ const units = ["cup", "gram", "ounce", "teaspoon"];
 --fcc-editable-region--
 const updateResultsList = () => {
   resultList.innerHTML = "";
+}
 
+recipeForm.addEventListener("submit", (event) => {
+  event.preventDefault();
+  updateResultsList();
+});
```

Comment:
aren't you asking to add this in this step? the learner should add this, it should not be in the seed code